packages/backend/src/graphql/schema.graphql

```diff
@@ -254,6 +256,15 @@ type Flow {
   steps: [Step]
   createdAt: String
   updatedAt: String
+  config: FlowConfig
+}
+
+type FlowConfig {
```

Comment:
nit: might be a good idea to put a comment saying that the rate limiting config (e.g. `maxQps`) is omitted